pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[tool.poetry]
+name = "mm_image_processing"
+version = "0.1.0"
```

Comment:
There are now two sources of truth! here and `mm_image_processing/__init__.py`.  This is not ideal

Reply:
Updated names.